internal/services/cosmos/cosmosdb_cassandra_table_resource.go

```diff
@@ -60,11 +60,13 @@ func resourceCosmosDbCassandraTable() *pluginsdk.Resource {
 			},
 
 			"analytical_storage_ttl": {
-				Type:         pluginsdk.TypeInt,
-				Optional:     true,
-				ForceNew:     true,
-				Default:      -2,
```

Comment:
THis is a breaking change for existing users? and would need to become copmuted

Reply:
@katbyte , it wouldn't cause breaking change because actually TF doesn't set the default value "-2" to request body based on [existing code](https://github.com/hashicorp/terraform-provider-azurerm/blob/f47f120d134e74bb38634a1b677fd4ca0377baf5/internal/services/cosmos/cosmosdb_cassandra_table_resource.go#L126).